backend/typescript/models/scheduling.model.ts

```diff
@@ -1,40 +1,57 @@
-import { Column, DataType, Model, Table } from "sequelize-typescript";
+import {
+  Column,
+  DataType,
+  Model,
+  Table,
+  AllowNull,
+  ForeignKey,
+  BelongsTo,
+  AutoIncrement,
+  PrimaryKey,
+} from "sequelize-typescript";
 import { Status } from "../types";
+import Donor from "./donor.model";
 
 @Table({ tableName: "scheduling" })
 export default class Scheduling extends Model {
-  @Column({ type: DataType.INTEGER, primaryKey: true })
-  id!: number;
-
-  // TODO: donor_id should be made a foreign key
-  // reference to the donors table once it is created
+  @PrimaryKey
+  @AutoIncrement
   @Column({ type: DataType.INTEGER })
-  donor_id!: string;
+  id!: number;
 
   @Column({ type: DataType.TEXT })
   description!: string | null;
 
   @Column({ type: DataType.INTEGER })
   quantity!: number | null;
 
```

Comment:
Can we also add a new field called `size`? This would correspond to how much space that donation would require in the fridge. This would again be an optional field. I will update the architecture doc with these new information. We might have to delete quantity later on, but I will clarify this with the client and update if we are keeping quantity or not.